### PR TITLE
Enable tests

### PR DESCRIPTION
**Pull request by alpkn**
Enable segfaulting getAvailableDevices, ExecGraphSerializationTest.ExecutionGraph_CPU after segfaults are fixed
#43063

### CODE REVIEW

**Review by ilyachur — APPROVED** [review]

**Comment by ilyachur** [comment]
@alpkn I will close this PR because PR https://github.com/openvinotoolkit/openvino/pull/3097 which contains the same changes was merged

**Comment by ilyachur** [comment]
Sorry my mistake. I re-open this PR.

**Review by iefode — APPROVED** [review]

File: inference-engine/tests/functional/inference_engine/ir_serialization/exec_graph.cpp [hunk 1]
```diff
@@ -109,7 +109,7 @@ class ExecGraphSerializationTest : public ::testing::Test {
     }
 };
 
-TEST_F(ExecGraphSerializationTest, DISABLED_ExecutionGraph_CPU) {
+TEST_F(ExecGraphSerializationTest, ExecutionGraph_CPU) {
```
Comment (iefode) [inline inference-engine/tests/functional/inference_engine/ir_serialization/exec_graph.cpp:112]: Why this tests is in `inference_engine`? I offer to move it to CPU func tests

**Review by iefode — CHANGES_REQUESTED** [review]

File: inference-engine/tests/functional/inference_engine/ir_serialization/exec_graph.cpp (hunk 1 above)
Comment (ilyachur) [reply inference-engine/tests/functional/inference_engine/ir_serialization/exec_graph.cpp:112]: I propose to solve this question separately. The current PR only enables some tests.

Comment (alpkn) [reply inference-engine/tests/functional/inference_engine/ir_serialization/exec_graph.cpp:112]: The issue of moving the test is not connected to this PR.
This code change is covered by https://github.com/openvinotoolkit/openvino/pull/3101
@ilyachur 
Please go ahead with that change.

**Pull request merged**